templates/definition/meter/iammeter.yaml

```diff
@@ -0,0 +1,82 @@
+template: iammeter
+products:
+  - brand: IAMMETER
+    description:
+      generic: WEM3080T/WEM3046T/WEM3050T
+  - brand: IAMMETER
+    description:
+      generic: WEM3080
+requirements:
+  description:
+    de: |
+      Die 3-phasigen Zähler (WEM3080T/WEM3046T/WEM3050T) benötigen die Aktivierung des Net Metering Mode (NEM) (=phasensaldierende Zählung).
+      Siehe https://www.iammeter.com/newsshow/net-energy-metering
+    en: |
+      3-phase meters (WEM3080T/WEM3046T/WEM3050T) require Net Metering Mode (NEM) to be enabled.
+      See https://www.iammeter.com/newsshow/net-energy-metering
+params:
+  - name: usage
+    choice: ["grid", "pv", "charge"]
+  - name: host
+render: |
+  type: custom
+  # net metered power/saldierte Leistung (W)
+  power:
+    source: http
+    uri: http://{{ .host }}/api/monitorjson
+    jq: if .Datas then .Datas[3][2] else .Data[2] end
+    cache: 2s
+  # net metered forward active energy/saldierter Energieverbrauch (kWh)
+  energy:
+    source: http
+    uri: http://{{ .host }}/api/monitorjson
+    jq: if .Datas then .Datas[3][3] else .Data[3] end
```

Comment:
last question out of curiosity: is the `if` really an "if exists" or doesn't that check for >0? The former would be correct, the latter not.

Reply:
I took a JSON response from the 1-phase and the 3-phase meters from [here](https://www.iammeter.com/newsshow/energy-meter-json-value) and ran the jq commands against them using [jq playground](https://play.jqlang.org/). The results have been as expected.

I can only test the 3-phase meter in real life though.

Reply:
Still seems this is incorrect

Reply:
Ah, nvm. Seems this is correct, sorry for the noise!